Create local extension sandbox (#691)
cursorless-dev/cursorless: docs/contributing/CONTRIBUTING.md

- [Git](https://git-scm.com/)
- [Node.js](https://nodejs.org/en/)
- [yarn](https://yarnpkg.com/)

### Steps

1. Clone [`cursorless`](https://github.com/cursorless-dev/cursorless)
2. Open the newly created `cursorless` directory in VSCode
3. Run the following in the terminal:

   ```bash
   yarn

>    yarn build

Reviewer: ```suggestion
   yarn esbuild
```

   yarn init-launch-sandbox
   ```

>    The last command creates a local sandbox with only the extensions that will be run alongside Cursorless when debugging locally. Please file an issue if you'd like to use other extension alongside Cursorless when running it locally for debugging

Reviewer: ```suggestion
   The `yarn init-launch-sandbox` command creates a local sandbox and only the extensions specified that will be run alongside Cursorless. Please file an issue if you'd like to use additional extensions when debugging locally.
```

Author: Yeah it runs its own set of extensions, so there should be no interaction with your locally installed extensions
